lib/relooper/src/graph/supergraph.rs

```diff
@@ -117,7 +117,7 @@ impl<TLabel: CfgLabel + Debug> SuperGraph<TLabel> {
     /// We are using that order for traversing supernodes graph for choosing between merge/split actions
     fn snode_order(&self) -> Vec<SLabel<TLabel>> {
         let start = self.nodes.get(&self.cfg.entry).unwrap().clone();
-        let res: Vec<_> = dfs_post_comparable(start.head, |slabel| {
+        let res: Vec<_> = DfsPost::<_, _, HashSet<_>>::reverse(start.head, |slabel| {
```

Comment:
The `into_iter().collect()` on lines 129-130 are notneeded because the `reverse` method returns a `Vec` already. This was true before since `dfs_post_comparable` also returned a `Vec`, but I didn't notice it until now. I can't comment on L129 directly because it's outside the diff.

Reply:
fixex